terraform/grafana-alerts/contact-points.tf

```diff
@@ -0,0 +1,38 @@
+resource "grafana_contact_point" "discord_channel_oracle_relayers_staging" {
+  name = "Discord #🚨|stg-oracle-relayers"
+
+  discord {
+    url     = var.discord_alerts_webhook_url_staging
+    title   = local.oracle_relayer_alert_config.title
+    message = local.oracle_relayer_alert_config.message
+  }
+}
+
+resource "grafana_contact_point" "discord_channel_oracle_relayers_prod" {
+  name = "Discord #🚨|prod-oracle-relayers"
+
+  discord {
+    url     = var.discord_alerts_webhook_url_prod
+    title   = local.oracle_relayer_alert_config.title
+    message = local.oracle_relayer_alert_config.message
+  }
+}
+
+# A catch all channel for all alerts and notification policies that don't have a specific contact point defined
+resource "grafana_contact_point" "discord_channel_catch_all" {
+  name = "Discord #alerts-catch-all"
```

Comment:
I like this, but it seems to be duplicating some of the legacy clients alerts at the moment?

Reply:
yup, turns out some of them were missing the `service=oracles` label, exactly the kind of thing that happens when managing stuff manually in the UI vs via terraform as code with PRs :)

I fixed them manually now by adding `service=oracles` everywhere I could find (and deleting some baklava rules)